### PR TITLE
147 BE getCategories

### PR DESCRIPTION
**Pull request by oriolsastre**
/api/v1/categories with a GET method now returns an array with all the categories (React, Javascript, Angular, etc.) as an array like this:
```
[
  {
    id: String,
    name: String
  }
]
```

### CODE REVIEW

**Comment by kevinmamaqi** [comment]
> /api/v1/categories with a GET method now returns an array with all the categories (React, Javascript, Angular, etc.) as an array like this:
> 
> ```
> [
>   {
>     id: String,
>     name: String
>   }
> ]
> ```

Swagger is ready?

**Comment by kevinmamaqi** [comment]
Hay conflictos @oriolsastre

File: back/src/openapi/routes/categories.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { registry } from '../registry'
+import { z } from '../zod'
+
+
+const pathRoot = '/api/v1/categories'
```
Comment (kevinmamaqi) [inline back/src/openapi/routes/categories.ts:5]: Esto debería estar en constantes y compartirse entre openapi y estas rutas

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]

File: back/src/openapi/routes/categories.ts (hunk 1 above)
Comment (oriolsastre) [reply back/src/openapi/routes/categories.ts]: /api/v1 has be stored as constant in appConfig, and now all routes and openapi use it from there.

**Comment by oriolsastre** [comment]
All paths are now defined in routes.ts, and used by routes and openapi from there.

**Pull request merged**